# This patch file was generated by NetBeans IDE
# Following Index: paths are relative to: /home/skosmalla/dev/NetBeansProjects/maven-dependency-analyzer
# This patch can be applied using context Tools: Patch action on respective folder.
# It uses platform neutral UTF-8 encoding and \n newlines.
# Above lines and this line are ignored by the patching process.
Index: src/main/java/org/apache/maven/shared/dependency/analyzer/DefaultProjectDependencyAnalyzer.java
--- src/main/java/org/apache/maven/shared/dependency/analyzer/DefaultProjectDependencyAnalyzer.java Base (BASE)
+++ src/main/java/org/apache/maven/shared/dependency/analyzer/DefaultProjectDependencyAnalyzer.java Locally Modified (Based On LOCAL)
@@ -131,7 +131,7 @@
 
     // private methods --------------------------------------------------------
 
-    private Map<Artifact, Set<String>> buildArtifactClassMap( MavenProject project )
+    protected Map<Artifact, Set<String>> buildArtifactClassMap( MavenProject project )
         throws IOException
     {
         Map<Artifact, Set<String>> artifactClassMap = new LinkedHashMap<Artifact, Set<String>>();
